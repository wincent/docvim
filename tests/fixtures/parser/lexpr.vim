" Bang form.
lexpr! system(&grepprg . ' ' . g:command)

" Vanilla.
lexpr s:MyFancyFunc()

" We support the abbreviation too.
lex autoloaded#function("blah").
