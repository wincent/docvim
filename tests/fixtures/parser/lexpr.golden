Project
  [ Project
      [ Unit
          [ LexprStatement
              { lexprBang = True
              , lexprExpr = "system(&grepprg . ' ' . g:command)"
              }
          , LexprStatement
              { lexprBang = False , lexprExpr = "s:MyFancyFunc()" }
          , LexprStatement
              { lexprBang = False
              , lexprExpr = "autoloaded#function(\"blah\")."
              }
          ]
      ]
  ]
